docs/debug.md

```diff
@@ -195,25 +184,23 @@ get the debug snap.
 
 ## Testing a service
 
-To test a service it must be installed first. See the section "Testing a
-binary" for the various ways to do that. Once it is installed the
-`snappy service status` command can be used to see if the service starts and
-runs as expected. The `snappy service logs` command is available to inspect
-the messages that the service send to `stdout`/`stderr`.
+To test a service it must be installed first. See the section "Testing a binary"
```

Comment:
I'd rather not-- it stretches markdown and I'm worried the automatic import will barf on it.